app/views/layout.html

```diff
@@ -5,10 +5,25 @@
 {{>includes/analytics}}
 {{/head}}
 
+{{$globalHeaderText}}GOV.UK{{/globalHeaderText}}
+{{$insideHeader}}
+
+{{#customBrandingImagePath}}
+<div class="header-logo header-logo--custom">
+    <a href="{{$homepageUrl}}https://www.gov.uk{{/homepageUrl}}" title="" id="logo" class="content">
+        <img src="{{customBrandingImagePath}}" class="custom-branding-image" alt=""> {{$globalHeaderText}}{{/globalHeaderText}}
+    </a>
+</div>
+{{$fontsCSS}}
+{{/fontsCSS}}
+{{/customBrandingImagePath}}
+{{/insideHeader}}
+
 {{$propositionHeader}}
 {{>includes/propositional_navigation}}
 {{/propositionHeader}}
 
+{{$bodyClasses}}custom-branding{{/bodyClasses}}
```

Comment:
@kakumara This is the line that is causing issues I think. this needs to be conditional